rubygem-safe_yaml/rubygem-safe_yaml.spec

```diff
@@ -0,0 +1,87 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name safe_yaml
+
+Summary: Parse YAML safely
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.4
+Release: 1%{?dist}
+Group: Development/Languages
+License: MIT
+URL: https://github.com/dtao/safe_yaml
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}ruby
+
+%if "%{?scl_ruby}" == "ruby193" || (0%{?el6} && 0%{!?scl:1})
+Requires: %{?scl_prefix_ruby}ruby(abi)
+BuildRequires: %{?scl_prefix_ruby}ruby(abi)
+%else
+Requires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+%endif
+
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildArch: noarch
+Provides: %{?scl_prefix_ruby}rubygem(%{gem_name}) = %{version}
```

Comment:
This should be scl_prefix only, so it provides `tfm-rubygem(..)`, not `ruby193-rubygem(..)`